src/main/java/net/sf/jabref/external/DroppedFileHandler.java

```diff
@@ -129,8 +130,8 @@ public void handleDroppedfile(String fileName, ExternalFileType fileType, MainTa
      *                  to a local temporary file.
      * @param entry     The target entry for the drop.
      */
-    public void handleDroppedfile(String fileName, ExternalFileType fileType, BibEntry entry) {
-        NamedCompound edits = new NamedCompound(Localization.lang("Drop %0", fileType.getExtension()));
+    public void handleDroppedfile(String fileName, Optional<ExternalFileType> fileType, BibEntry entry) {
```

Comment:
It just seemed to make sense to use Optional as default instead of using Optional.of and get() just to pass ExternalFileType around (since all(?) ExternalFileTypes are obtained as Optionals now). For these two methods one may just as well do the get() in the previous method (since the ExternalFileType is obtained from one of the ExternalFileTypes methods, returning an Optional, which is already checked for content). Not sure if it matters.